services/av_worker.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file av_worker.js
+ * @author Capuccino
+ * @license BSD-3-Clause
+ */
+global.Promise = require("bluebird");
+const micro = require("micro");
+const https = require("https");
+const {json, send} = micro;
+const config = require("./av.config");
+
+const server = micro(async req => {
+    const data = json(req);
+
+    request("POST", `https://www.virustotal.com/vtapi/v2/url/scan?apikey=${config.apiKey}`, {}, {data}).then(res => {
+
+        if (res.statusCode === "204") send(res, 204, JSON.stringify({code: "204", reason: "Exceeded VirusTotal Ratelimit."}));
```

Comment:
VirusTotal returns a 204 for ratelimiting which is dumb IMO, but we should return a 429 instead, as that's the standard.